java/src/org/openqa/selenium/PersistentCapabilities.java

```diff
@@ -58,7 +56,7 @@ public PersistentCapabilities setCapability(String name, Object value) {
   @Override
   public Map<String, Object> asMap() {
     return getCapabilityNames().stream()
-        .collect(toUnmodifiableMap(Function.identity(), this::getCapability));
+        .collect(Collectors.toUnmodifiableMap(Function.identity(), this::getCapability));
```

Comment:
This change makes sense. But these are unrelated to changes made as part of the rest of this PR. Can we please separate this out into a separate PR?

Reply:
That's fine, I'll create a separate PR for these changes.